edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.CloudProxy.Test/EventHubReceiver.cs

```diff
@@ -4,52 +4,39 @@ namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy.Test
     using System;
     using System.Collections.Generic;
     using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Common;
     using Microsoft.Azure.EventHubs;
 
     public class EventHubReceiver
     {
-        readonly EventHubClient eventHubClient;
+        const string EventHubConsumerGroup = "ci-tests";
+        readonly string eventHubConnectionString;
```

Comment:
why remove EventHubClient and store eventHubConnectionString instead?

Reply:
If we are going to retry the GetMessagesForDevice method, then we should start the process from the beginning, which means creating the EventHubClient again. Do you agree?